inception-doc/src/main/resources/META-INF/asciidoc/user-guide/getting-started.adoc

```diff
@@ -1,86 +1,412 @@
+
+
+*Create your first annotations*
+
+You may want to watch our https://www.youtube.com/watch?v=wp4AN3p23mQ&list=PL5Hz5pttaj96SlXHGRZf8KzlYvpVHIoL-&index=3&t=0s[tutorial video „Overview“] on how to create
+annotations. But this guide, too, will lead you step by step. We will create a Named Entity annotation which tells whether a mention is a person (PER), location (LOC), organization (ORG) or other (OTH):
+
+NOTE: *Creating an Own Project:* In this guide, we will use our example project. If you would like to create an own project later on, click on *create*, enter a project name and click on *save*. Use the *Projects* link at the top of the screen to return to the project overview and select the project you just created to work with it. See <<Project Settings>> in order to add documents, users, guidelines and more to your project.
+
+*Step 1 - Opening a Project:* After logging in, what you see first is the *Project overview*. Here, you can see all the projects which you have access to. Right now, this will be only the example project. Choose the example project by clicking on its name and you will be on the *Dashboard* of this project.
+
+image:getting_started_open_a_project.png[image,width=758,height=179]
+
+
+NOTE: *Instructions to Example Projects:* In case of the example project, on the dashboard you also find instructions how to use it. This goes for all our example projects. You may use it instead of or in addition to the next steps of this guide. +
+In case of own projects, you will find the description you have given it instead.
+
+*Step 2 - Open the Annotation Page:* In order to annotate, click on *Annotation* on the top left. You will be asked to open the document which you want to annotate. For this guide, choose _pets1.tsv_.
+
+NOTE: *Annotations in Newly Imported Projects:* In the example project, you will see several annotations already. If you import projects or single documents (see xref:documents_in_getting_started[Documents]) without any annotations, there will be none. But in the
+example projects, we have added some annotations already as examples. If you export a project (see  xref:export_in_getting_started[Export]) and import it again (as we just did with the example project in <<Download and Import an Example Project>>), there will be the same annotations like before.
+
+*Step 3 - Create an Annotation:* After opening the document, select *Named entity* from the *Layer* dropdown menu on the right side of the screen to create your first annotation. Then, use the mouse to select a word in the annotation area, e.g. _in my home_ in line one. When you release the mouse button, the annotation will immediately be created and you can edit its details in the right sidebar (see next paragraph).
+
+*_Note:_* All annotations will be saved automatically without clicking an extra save-button. This is why there is none.
```

Comment:
I would remove This is why there is none. or This is why there is none save button in the annotation editor.